test/image-prepare

```diff
@@ -228,20 +229,21 @@ def main():
     if args.quick:
         customize.append("--quick")
 
-    dist_tar = subprocess.check_output([f'{BASE_DIR}/tools/make-dist'], text=True).strip()
+    if not args.no_c:
+        dist_tar = subprocess.check_output([f'{BASE_DIR}/tools/make-dist'], text=True).strip()
 
-    if args.image == "fedora-coreos":
-        customize += build_install_coreos(dist_tar, args.image, args.verbose, args.quick)
-    elif args.image.startswith("rhel-8") or args.image.startswith("centos-8"):
-        customize += build_install_rhel8(dist_tar, args.image, args.verbose, args.quick)
-    else:
-        customize += build_install_package(dist_tar, args.image)
+        if args.image == "fedora-coreos":
+            customize += build_install_coreos(dist_tar, args.image, args.verbose, args.quick)
+        elif args.image.startswith("rhel-8") or args.image.startswith("centos-8"):
+            customize += build_install_rhel8(dist_tar, args.image, args.verbose, args.quick)
+        else:
+            customize += build_install_package(dist_tar, args.image)
 
-    if not args.quick:
-        customize += validate_packages()
+        if not args.quick:
+            customize += validate_packages()
 
-    # post build/install test preparation
-    customize += ["--script", os.path.join(TEST_DIR, "vm.install")]
+        # post build/install test preparation
+        customize += ["--script", os.path.join(TEST_DIR, "vm.install")]
```

Comment:
This still needs to run for `--python`. Otherwise, if `--no-c` is always just meant to hack an existing image, then that's what `scp -r` is for, or `image-customize --upload`?

Reply:
Ya.  This is really for hacking an existing image (ie: with `-o`).  And indeed, that could be replaced by an `image-customize` invocation, but the logic for building the wheel is also in this script...

Maybe we need `tools/make-wheel` alongside our `tools/make-dist`.  But that's sort of what I was trying to avoid.

Reply:
OK, it seems you have a very different way of updating images for iterating than I (I really just use `scp -r src/cockpit ..`).

Reply:
I'm happy to drop the `--no-c` commit (although I think the other one still makes sense).